### DISCUSSION TITLE
[Office Hours] Creating an unfixed number of view duplicates

### DISCUSSION THREAD

**Discussion by kc224** [CS342/.github#87] [Office Hours]
### Related Discussion

This [branch](https://github.com/CS342Industry/2024-Nourish/tree/new-onboarding-videos)

### Reproduction

After a user fills out their onboarding questionnaire, I need to fetch the response codes and display the corresponding videos during the onboarding process. I'm having trouble fetching the correct response codes and also implementing a variable number of video screen views.

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by kevinawang** [comment]
I am also having trouble fetching data from firebase! Ina and I are on the same team so I will join her on this question.
